ISSUES.md

```diff
@@ -0,0 +1,31 @@
+# Issue triaging
+
+## `needs author feedback`
+
+This is something that needs additional clarification from the author.
+
+Note: issues with this label are automatically marked stale if there is no activity for 7 days,
+and then closed automatically if there is no response from the author within 7 days after that.
+
+## `needs minimal repro`
+
+Minimal repros are helpful for a lot of issues (not only for bugs).
+
+They allow the maintainers (and others) to much better understand exactly what the
+issue author is trying to do, and what they expect to happen.
+
+Ideally minimal repros should be posted to a github repository,
+and should include a `README.md` that explains exactly how to build and run the repro,
+and what the output of the repro is, and what you expect (or would like) the output of
+the repro to be.
+
+Note: issues with this label are automatically marked stale if there is no activity for 30 days,
+and then closed automatically if there is no response from the author within 30 days after that.
+
+## `needs external contribution`
```

Comment:
thinking to rename `contribution welcome` to this (I like the `needs ...` meaning it needs something to become unblocked)

Reply:
I understand the desire for consistency, but this reads less community/collaboration and rather more like an us-or-them separation.